geomesa-tools/examples/ingest/example1.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+geomesa ingest -u user -p password -i inst -z zoo -c catalog -conf example1.conf example1.csv
```

Comment:
yeah it supports --conf we (esp me) have been bad about single dash args with >1 length...perhaps we should dial that back? we use -c for catalog... we could also support just the single dash arg or a capital -C 

see https://github.com/locationtech/geomesa/blob/master/geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/IngestCommand.scala#L43